### ISSUE TITLE
Option to combine tabs and title bar

### ISSUE THREAD

**Issue by akbyrd** [microsoft/vscode#107123]
I generally use VS Code in fullscreen mode to maximize the vertical space available for code. However, I also frequently have to use it as a regular window so I can see something next to it, such as a debugger or other tool. I'd love to be able to reclaim the extra space I get in fullscreen mode even when in a window.

Specifically, I'm interested in an option to combine editor tabs and the title bar. Literally just wedge them together. I don't need the buttons on the right side of the tab region, so this actually looks fairly clean, imo.

Current
![image](https://user-images.githubusercontent.com/484784/93721703-b0c1e100-fb46-11ea-8e09-1448f32c65ec.png)

Desired
![image](https://user-images.githubusercontent.com/484784/93721825-6c831080-fb47-11ea-844b-3a069f7453fb.png)

Honestly, don't really need the logo either
![image](https://user-images.githubusercontent.com/484784/93721895-cedc1100-fb47-11ea-90cd-bf28313f6086.png)

**Comment by akbyrd** [comment]
Looks like there are similar requests for [Mac](https://github.com/microsoft/vscode/issues/12377) and [Linux](https://github.com/microsoft/vscode/issues/43154) with quite a lot of interest.

**Comment by akbyrd** [comment]
The original pitch for https://github.com/microsoft/vscode/issues/17060 was to merge the title and tabs as well. 316 votes between the Linux, OSX, and Windows requests. 406 if you count the 17060.

**Comment by detzt** [comment]
This is by far my most missed feature of VS Code. However for me the main benefit would be switching tabs by clicking on the topmost pixel of the monitor, which makes it a lot faster and reliable to click (see Fitt's Law and the UI of all modern browsers).

**Comment by mpekurin** [comment]
An image from an old issue (#71777) with the same suggestion, shows how it may look with all UI visible:
![55616056-a4eb2f00-5799-11e9-8aec-fc62f7c5b128](https://user-images.githubusercontent.com/22733346/141768136-334b3bae-2491-4da2-9bde-cf898b2f4aa7.png)
Missing vscode logo or a hamburger for the menu bar. Just imagine it's here.

**Comment by YushArora** [comment]
Any update?

**Comment by ferssor** [comment]
Any update? we can vote this:

**Comment by ferssor** [comment]
Any update? We can vote this?

**Comment by dezren39** [comment]
Is there any progress on this? It's a pretty common feature at this point, in windows terminal and all over. There's even old examples with extensions and that one fork. Vertical space is very important. It's nice having the menu bar in the title bar, but I would rather have tabs, there is a keyboard shortcut for the menu bar.

**Comment by backermanbd** [comment]
I am in search of the same feature.

**Comment by napuzu** [comment]
https://github.com/microsoft/vscode/assets/88090558/5c36c35a-6c05-4973-861a-84c6296a5906

Just a bug, and then it changes back again, I hope it's not.

**Comment by github-account1111** [comment]
> It's nice having the menu bar in the title bar, but I would rather have tabs, there is a keyboard shortcut for the menu bar.

Yeah, at least integrate the tab bar into the title bar when the menu bar is unchecked in View, which hides the menu under a burger button and results in a virtually empty title bar:
	<img width="1217" height="97" alt="Image" src="https://github.com/user-attachments/assets/081aaeed-b03d-4d55-80d1-1b8d23716c2b" />
